server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -269,17 +267,25 @@ export function configureWebSocketServices(
 		async function connectDocument(message: IConnect): Promise<IConnectedClient> {
 			const startTime = Date.now();
 			const throttleErrorPerCluster = checkThrottleAndUsage(
-				connectThrottlerPerCluster,
+				throttlersMap?.get("generalCluster")?.get("socketConnections"),
 				getSocketConnectThrottleId("connectDoc"),
 				message.tenantId,
 				logger,
 			);
 			if (throttleErrorPerCluster) {
 				return Promise.reject(throttleErrorPerCluster);
 			}
+
```

Comment:
This code is getting complicated with each iteration. :D It would be great if you could add some code comments to make it readable for other developers?

For example here, you can add something like:

If tenant is found in any tenantGroup list, apply tenantGroup specific throttling limits, else apply general per tenant throttling limits.

Feel free to edit or add more details ^

Reply:
Thanks. Done. :)